### PR TITLE
Dev containers

### PR DESCRIPTION
**Pull request by sushant94**

### CODE REVIEW

File: src/frontend/containers.rs [hunk 1]
```diff
@@ -164,7 +164,7 @@ fn ssa_single_fn(f: &FunctionInfo,
 
 // This function fixes up the call information.
 //
-// During SSA construction, call instructions are special cased, i.e. we do not sue the
+// During SSA construction, call instructions are special cased, i.e. we do not use the
```
Comment (chinmaydd) [inline src/frontend/containers.rs]: Let's not fix this typo please

**Comment by chinmaydd** [comment]
What do you think about implementing the trait `ProjectLoader` for the `R2` struct directly? I don't see the point of having a separate struct(unless you have something in mind) and not add it to radare2-r2pipe-api.

**Comment by chinmaydd** [comment]
This looks good @sushant94. Are you waiting for arch-rs changes or I think we can be good to merge?

**Comment by sushant94** [comment]
@chinmaydd I'm adding in more fields and methods. You can use this branch though for now if you need it.

**Comment by sushant94** [comment]
These commits will currently not pass the tests as I've disabled some modules.
These commits contain some breaking changes as `ValueType`, one of the core structs in IR has changes. I'll add in more comments to rationalize these changes.

@ZhangZhuoSJTU The latest commits contain a major refactor, which may conflict with verfier.rs if you've made changes. I'll be re-enabling verifier.rs soon.

**Comment by ZhangZhuoSJTU** [comment]
@sushant94 That's Ok. I haven't done anything on verifier.rs. Give me some time to catch your progress.

**Comment by ZhangZhuoSJTU** [comment]
Would you mind merging thithis PR ? Because we may both work on SSA right now.  Or is there any good way to avoid conflicts (╥﹏╥)

**Comment by sushant94** [comment]
@ZhangZhuoSJTU Yes, I'm ready to merge the PR. Just wanted you to see this and catch up on the changes.

**Pull request merged**